src/setup-gazebo-macos.ts

```diff
@@ -4,16 +4,34 @@ import * as brew from "./package_manager/brew";
 /**
  * Tap into OSRF repository
  */
-async function addBrewRepo() {
+async function addBrewRepo(): Promise<void> {
 	await utils.exec("brew", ["tap", "osrf/simulation"]);
 }
 
+/**
+ * Overwrite existing python installation
```

Comment:
Could we please add here a comment with the reasoning of making this uninstall / install special steps.